pinot-server/src/main/java/org/apache/pinot/server/starter/grpc/DataTableHandler.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pinot.server.starter.grpc;
```

Comment:
We do have an existing class named DataTableHandler -- the Netty inbound handler on the broker side that receives the DataTable from server.

Although the GRPC handler is in a different package, you may want to consider naming differently (probably have grpc prefix or suffix).

Secondly, from the interface it looks like this is slightly different from our existing DataTableHandler. It looks like, this does both (1) (outbound) submission of request to server and (inbound) receiving the response which is DataTable. Is that correct?

It would be nice to add detailed Javadoc highlighting the semantics of this handler